bin/install.sh

```diff
@@ -35,14 +35,8 @@ grep '^#OktaAWSCLI' "${bash_functions}" > /dev/null 2>&1
 if [ $? -ne 0 ]
 then
 echo '
-function withokta {
-    java -Djava.net.useSystemProxies com.okta.tools.WithOkta $@
-}
 function aws {
-    withokta "aws --profile $1" $@
-}
-function sls {
-    withokta "sls --stage $1" $@
+    withokta aws $@
 }
```

Comment:
Good catch 👍

The existing **withokta** function is broken (my fault). It doesn't set the classpath properly. This should work:

```java
java -Djava.net.useSystemProxies -classpath ~/.okta/okta-aws-cli.jar com.okta.tools.WithOkta $@
```
I'm going to follow up with a PR with a correct **withokta** and **sls** function based on your approach.

Reply:
Thanks for merging!  My only question on that is why bother with this bash function when the `withokta` in /usr/local/bin works just fine?  Seems to create a confusion and multiple places that need updating if the underlying .jar changes.

Reply:
You are correct. It already creates the command as a file so the function is not required.

Reply:
Cool, thanks!